File: 50 Coding Challenge Part I/JavaScript/Challenge9.js
/**
 * Challenge 9:
 * Create a function that will convert from  Fahrenheit to Celsius
 * Reminder: C = F – 32 / 1.8
 */
function fahrenheitToCelsius(tem){
    return (tem - 32) / 1.8;
}

console.log(fahrenheitToCelsius(10).toFixed());

File: 50 Coding Challenge Part I/Python/Challenge9.py
"""
Challenge 9: Create a function that will convert from Fahrenheit it to Celsius
"""
def fahrenheit_to_celsius(i):
    return (i + 32) * 1.8


tem = int(input("Enter temperature in Fahrenheit: "))
print(round(fahrenheit_to_celsius(tem), 1))
